MAUI/ImageEditor/getting-started.md

```diff
@@ -0,0 +1,131 @@
+
+4. Initialize `SfImageEditor.`
+
+{% tabs %}
+{% highlight XAML hl_lines="3 5" %}
+
+<ContentPage   
+    . . .
+    xmlns:imageEditor="clr-namespace:Syncfusion.Maui.ImageEditor;assembly=Syncfusion.Maui.ImageEditor">
+
+    <imageEditor:SfImageEditor Source="image_name.extension" />
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# hl_lines="1 9 10" %}
+
+using Syncfusion.Maui.ImageEditor;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfImageEditor imageEditor = new SfImageEditor();
+        this.Content = imageEditor;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+using Syncfusion.Maui.Core.Hosting;
+namespace GettingStarted
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+
+            builder.ConfigureSyncfusionCore();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("Segoe-mdl2.ttf", "SegoeMDL2");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+##Loading an image to image editor
+
+Refer to the following steps to add an image to the pcl project:
+
+1. Locate the "Resources" folder in your .NET MAUI project. This folder is typically located in the project's root directory.
+2. Right-click on the "Resources" folder in the project structure view or Solution Explorer.
+3. From the context menu, select "Add" and then "Existing Item." This will open a file selection dialog.
+4. Browse to the location on your computer where the image file is stored.
+5. Select the image file you want to add to the "Resources" folder.
+6. Click the "Add" button to add the image file to the project.
+7. After the image has been imported, ensure whether the image Build Action has been set to `EmbeddedResource`.
+
+N> Image formats such as JPEG and PNG can be loaded to the image editor.
+
+The following code shows adding an image to the image editor control with the format as "JPEG" and name as "image".
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:imageEditor="clr-namespace:Syncfusion.Maui.ImageEditor;assembly=Syncfusion.Maui.ImageEditor"
+             x:Class="Syncfusion.MainPage">
+
+             <imageEditor:SfImageEditor Source="syncfusion.png"/>

```

Comment:
Source="syncfusion.png" as per headline, this should be

Source="image.jpeg"

Reply:
image name and extensions are changed